sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -237,6 +237,9 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
         Nil
       }
 
+    case DescribeTable(catalog, ident, _, isExtended) =>
+      DescribeTableExec(catalog, ident, isExtended) :: Nil
```

Comment:
how about
```
case DescribeTable(catalog, ident, r: DataSourceV2Relation, isExtended) =>
  DescribeTableExec(r.table, ident, isExtended) :: Nil
```
Then we don't need to lookup the table again in `DescribeTableExec`